lib/api/client-commands/pause.js

```diff
@@ -26,10 +25,8 @@ util.inherits(Pause, events.EventEmitter);
 
 Pause.prototype.command = function(ms, cb) {
   var self = this;
-  // If we don't pass the milliseconds, the client will
-  // be suspended indefinitely
   if (!ms) {
-    return this;
+    throw new Error('method expects milliseconds argument.');
```

Comment:
@beatfactor I tried to throw an error here, but the test isn't catching it. Is there something I'm missing?